cli/src/main.rs

```diff
@@ -189,16 +189,32 @@ fn main_inner() -> Result<(), String> {
             )?;
         }
         ("global", Some(global_args)) => {
+            /// Do less verbose output if quiet-flag is set and return status code
+            fn quiet_output_handle(result: bool, quiet_flag: bool) {
+                if result {
+                    if !quiet_flag {
+                        println!("Model satisfies formula: {}", result);
+                    }
+                    std::process::exit(42);
+                } else {
+                    if !quiet_flag {
+                        println!("Model satisfies formula: {}", result);
+                    }
+                    std::process::exit(43);
```

Comment:
Remove the duplicate code by moving the print outside the outer if.

Reply:
Fixed in e6e2663